docs/user/getting-started.asciidoc

```diff
@@ -1,55 +1,66 @@
 
-[float]
+. Log into the link:https://cloud.elastic.co/[Elasticsearch Service Console].
+If you need an account, register for a link:https://www.elastic.co/cloud/elasticsearch-service/signup[free 14-day trial].
+
+. Click *Create deployment*, then give your deployment a name.
 
-=== Add data tutorials
-{kib} has built-in *Add Data* tutorials to help you set up
-data flows in the Elastic Stack. These tutorials are available 
-from the Kibana home page. In *Add Data to Kibana*, find the data type 
-you’re interested in, and click its button to view a list of available tutorials. 
+. To use the default options, click *Create deployment*. If you want, you can modify the other deployment options, but the default options are great to get started.
+
+Be sure to copy down the password for the `elastic` user and Cloud ID information. You'll need that later.
 
 [float]
-=== Hands-on experience 
+[[get-data-in]]
+== Get data into {kib}
 
-The following tutorials walk you through searching, analyzing, 
-and visualizing data.
+Before you create visualizations and analyze your data, you must get the data into {kib}.
 
-* <<tutorial-sample-data, Explore Kibana using sample data>>. You'll
-learn to filter and query data, edit visualizations, and interact with dashboards. 
+The easiest way to get data into {kib} is to add a sample data set. {kib} has several sample data sets that you can use before loading your own data.
 
-* <<tutorial-build-dashboard, Build your own dashboard>>.  You'll manually load a data set and build 
-your own visualizations and dashboard. 
+These sample data sets showcase a variety of use cases:
 
-[float]
-=== Before you begin
+* *Sample eCommerce orders* includes visualizations for tracking product-related information,
+such as cost, revenue, and price.
 
-Make sure you've <<install, installed Kibana>> and established
-a <<connect-to-elasticsearch, connection to Elasticsearch>>.
+* *Sample flight data* includes visualizations for monitoring flight routes.
 
-If you are running our https://cloud.elastic.co[hosted Elasticsearch Service] 
-on Elastic Cloud, you can access Kibana with a single click.
+* *Sample web logs* includes visualizations for monitoring website traffic.
 
+To use the sample data sets:
 
---
+. Go to the {kib} home page.
+
+. Click *Load a data set and a {kib} dashboard*.
+
+. Click *View data* and view the prepackaged dashboards, maps, and more.
+
+[role="screenshot"]
+image::images/add-sample-data.png[]
 
-include::{kib-repo-dir}/getting-started/add-sample-data.asciidoc[]
+NOTE: The timestamps in the sample data sets are relative to when they are installed.
+If you uninstall and reinstall a data set, the timestamps change to reflect the most recent installation.
+
+[float]
+[[getting-started-next-steps]]
+== Next steps
+
+* To get a hands-on experience using the sample flight data, follow the <<tutorial-sample-data, add sample data>> tutorial.
+
```

Comment:
Maybe here add a bullet point about the add data tutorial

* To quickly set up a data flow in the Elastic Stack, use our Add data tutorials, available from the home page.